molecule/default/verify.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Check drush is installed
+  hosts: all
+  become: yes
+
+  tasks:
+    - name: Confirm drush installation is correct.
+      shell: drush --version | grep '8.1.17'
+      changed_when: false
```

Comment:
I think it's OK to have non-idempotent tasks in verify. Afaik it's just the main 'converge' file that gets tested for idempotency.